### PR TITLE
Update cards-format.md

### PR DESCRIPTION
**Pull request by ojasvichoudhary**
add support for full width adaptive cards

### CODE REVIEW

File: msteams-platform/task-modules-and-cards/cards/cards-format.md [hunk 1]
```diff
@@ -152,6 +152,37 @@ To include a mention in an Adaptive Card your app needs to include the following
 }
 ```
 
+### Full width Adaptive Cards
+You can use this proprety to expand the width of an adaptive card and make use of addtional canvas space (see example below).
+
+#### Constructing full width cards
+To make a full width Adaptive Card your app needs to include the following elements
+
+* The `width` object inside of an `msteams` property in the card content should be set to `Full`
```
Comment (KirtiPereira) [inline msteams-platform/task-modules-and-cards/cards/cards-format.md:161]:
```suggestion
To make a full width Adaptive card the `width` object in `msteams` property in the card content must be set to `Full`.
In addition, your app must include the following elements:
```

**Review by KirtiPereira — CHANGES_REQUESTED** [review]
@ojasvichoudhary - please verify and commit the suggestions.
Thank you.

**Comment by KirtiPereira** [comment]
@ojasvichoudhary - please incorporate the suggested changes, so we can merge the PR.
Thank you.

**Comment by surbhigupta12** [comment]
@ojasvichoudhary  - please incorporate the suggested changes, so we can merge the PR.
Thank you.

**Comment by KirtiPereira** [comment]
@ojasvichoudhary - have uploaded the screenshots as discussed. Please verify the placement is appropriate and also commit the suggestions / comments already provided and confirm if the PR can be merged.

**Comment by ojasvichoudhary** [comment]
@KirtiPereira changes look good, can you please push this PR out, I'll create a separate one for other cards changes.

**Review by ojasvichoudhary — COMMENTED** [review]
changes look good

**Review by KirtiPereira — APPROVED** [review]
Approved

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**